cmd/config/storageclass/storage-class.go

```diff
@@ -38,7 +38,7 @@ const (
 
 	// Valid values are "write" and "read-write"
 	DMAWrite     = "write"
-	DMAReadWrite = "read-write"
+	DMAReadWrite = "read+write"
```

Comment:
This or comment above is wrong.

Reply:
I changed it let me fix the comment as well.